app/models/user.rb

```diff
@@ -33,7 +33,7 @@
 #
 
 class User < ActiveRecord::Base
-  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled
+  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled, :role, :profile_attributes
```

Comment:
hi, could you explain why you've added `role` and `profile_attributes` here?  Maybe I'm missing something but I can't see them being used.